<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SomethingLikeScapy</class>
 <widget class="QMainWindow" name="SomethingLikeScapy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="summary">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>241</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Source IP</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Source Port</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Protocol</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Destination IP</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Destination Port</string>
     </property>
    </column>
   </widget>
   <widget class="QTreeWidget" name="analysis">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>541</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sniffStop">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOnlineSniff"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionOnlineSniff">
   <property name="text">
    <string>OnlineSniff</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
